bring-it-up/bring-it-up
# Added hours to counselling services

## Description
#### Summary:
- Added hours data (opening and closing times) to counselling services

#### Changes:
- Added hours to counselling service schema
- hours is an object, each key has an array value, that array can hold either one or two elements, if it holds one, that element can be either 0 (closed) or 1 (All-day), otherwise times are represented as 24-hour floats (e.g. 8:30 PM = 20.3), keys are first 3 letters of day (e.g. Monday = mon)
- Added request validators for hours data
- Updated data in counselling-services database to include hours
- Updated the API docs to include hours in the Query Params and Example response

#### How to test this PR:
- To test the validators for hours run unit tests
- Integration tests are in the `server/tests/unit` directory
- Run the integration tests: `npm run test:unit`
- You can also manually go to 'http://localhost:4000/counselling-services' and check if hours data shows up
- Use Postman to try manually posting valid or invalid collection with hours data
  
## Checklist

- [x] I ran all the tests and they were successful
